doc/api/crypto.md

```diff
@@ -2767,7 +2902,8 @@ crypto.randomFill(buf, 5, 5, (err, buf) => {
 });
 ```
 
-Any `TypedArray` or `DataView` instance may be passed as `buffer`.
+Any `ArrayBuffer` `TypedArray` or `DataView` instance may be passed as
```

Comment:
Yes, WebCrypto requires that ArrayBuffer be supported. Accordingly, it makes sense to support them also in the legacy API since the underlying code is going to need to support them.

Reply:
Note also that nearly all of the WebCrypto API promises that do not resolve as `CryptoKey` or `boolean` resolve as `ArrayBuffer`.